app/assets/stylesheets/views/_topics.scss

```diff
@@ -1,5 +1,12 @@
 .topics-page {
 
+  // Topic beta labels have a lot of text. Limiting the width of the label
+  // prevents the lines to wrap under the label.
+  .topic-beta-label .govuk-beta-label span {
+    width: 75%;
+    vertical-align: top;
+  }
```

Comment:
Ideally we shouldn't be styling the internals of a component within an application. If we did then we won't be able to iterate the component reliably without things breaking. Is there a way this could be incorporated into the components stylesheet?